src/mfs.jl

```diff
@@ -0,0 +1,215 @@
+using BlackBoxOptim: bboptimize, best_candidate
+using Random: GLOBAL_RNG
+using StateSpaceSets: statespace_sampler
+export minimal_fatal_shock, MFSBruteForce, MFSBlackBoxOptim
+
+
+
+"""
+    minimal_fatal_shock(mapper::AttractorMapper, u0, search_area, algorithm::Union{MFSBruteForce, MFSBlackBoxOptim})  
+
+Runs the minimal fatal shock algorithm on the initial point `u0`. Two algorithms are available: MFSBruteForce and MFSBlackBoxOptim.
+MFSBruteForce is an algorithm based on randomised search with two consequent steps: random initialization and sphere radius reduction.
+MFSBlackBoxOptim is an algorithm based derivative free optimization. It uses BlackBoxOptim package to find the best shock.
+
+`mapper::AttractorMapper` one of available in Attractors.jl [`AttractorMapper`](@ref) constructed with respect to dynamical system.
+`u0` vector containing coordinates of initial point to be tested.
+`search_area` array of tuples, by specifying single value e.g. `[(-1.5, 1.5)]` you specify this value for each of the variables, or you may specify separetely 
+search area for each of your states by, for example in 2 dimensions, `[(-1.5, 1.5), (-1, 1)]`.
+`algorithm` one of the two algorithms available: `MFSBruteForce` or `MFSBlackBoxOptim`.
+
+## `MFSBruteForce`
+`initial_iterations` number of random pertubations to try in the first step of the algorithm, default = 10000.
+`sphere_iterations` number of steps while initializing random points on hypersphere and decreasing its radius, default = 10000.
+`sphere_decrease_step` factor by which the radius of the hypersphere is decreased, default = 1000.0, the higher the value, 
+the more precise values may be obtained.
+
+## `MFSBlackBoxOptim`
+`guess` vector of initial guesses for the optimization algorithm, `default = []`.
+`MaxSteps` maximum number of steps for the optimization algorithm, default = 10000.
+`penalty` penalty value for the objective function, allows to adjust optimization algorithm to find the minimal fatal shock, `default = 1000.0`
+
+
+## Output
+`best_shock` - minimal fatal shock.
+`best_dist` norm of the minimal fatal shock.
```

Comment:
Output should be described in the first sentences immediatelly after the function call signature. It doesn't need a section.